droid-hal-device.inc

```diff
@@ -399,8 +399,12 @@ else
 fi
 
 # Hadk style build of android on OBS 
+if ubu-chroot -v ; then
+   sh="sh -c"
```

Comment:
LGTM, worth commenting that argument parsing has been introduced together with version reporting.